superset-websocket/src/config.ts

```diff
@@ -76,51 +78,48 @@ function configFromFile(): Partial<ConfigType> {
   const isTest = process.env.NODE_ENV === 'test';
   const configFile = isTest ? '../config.test.json' : '../config.json';
   try {
-    return require(configFile);
+    const configData = fs.readFileSync(configFile, 'utf8');
+    const config = JSON.parse(configData);
+    return config;
   } catch (err) {
-    console.warn('config.json file not found');
+    console.warn('Error loading config file:', err);
     return {};
   }
 }
 
-const isPresent = (s: string) => /\S+/.test(s);
-const toNumber = Number;
-const toBoolean = (s: string) => s.toLowerCase() === 'true';
-const toStringArray = (s: string) => s.split(',');
```

Comment:
I notice we're no longer using these. Is the assumption that they're already in correct format in the config file? If so that makes sense IMO, I'm not sure we need to support incorrectly formatted config parameters.